examples/closure/WORKSPACE

```diff
@@ -25,10 +25,50 @@ http_archive(
     urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/1.4.0/rules_nodejs-1.4.0.tar.gz"],
 )
 
+http_archive(
+    name = "rules_jvm_external",
+    sha256 = "79c9850690d7614ecdb72d68394f994fef7534b292c4867ce5e7dec0aa7bdfad",
+    strip_prefix = "rules_jvm_external-2.8",
+    url = "https://github.com/bazelbuild/rules_jvm_external/archive/2.8.zip",
+)
+
 load("@build_bazel_rules_nodejs//:index.bzl", "yarn_install")
 
 yarn_install(
     name = "npm",
     package_json = "//:package.json",
     yarn_lock = "//:yarn.lock",
 )
+
+load("@rules_jvm_external//:defs.bzl", "maven_install")
+
+# https://www.npmjs.com/package/google-closure-templates is a broken package
```

Comment:
wow it has also been a year since it was released
https://unpkg.com/browse/google-closure-templates@20190311.0.7/
they should really just take down the npm package since it's DOA